For/51.c

```diff
@@ -0,0 +1,17 @@
+// Super for aninhado
+#include <stdio.h>
+int main()
+{
+    int i, j, k;
+    for (i = 1; i <= 100; i++)
```

Comment:
quando voce utiliza <= (menor igual) ele faz duas comparações

toda vez que ele executa o for, ele verifica se o valor é menor e também se o valor é igual, pra aumentar a velocidade, normalmente usamos somente o sinal de menor, aumentando o numero do for, por exemplo
for(i = 1; i<101;i++)